metrics-proxy/src/main/java/ai/vespa/metricsproxy/http/ValuesFetcher.java

```diff
@@ -49,6 +49,10 @@ public ValuesFetcher(MetricsManager metricsManager,
                 .collect(Collectors.toList());
     }
 
+    public List<MetricsPacket> fetchAllMetrics() throws JsonRenderingException {
```

Comment:
Please update `fetch(String)` to use this method instead of using metricsManager directly, to avoid duplication.